core/templates/components/concept-card/concept-card.component.html

```diff
@@ -0,0 +1,84 @@
+<div class="oppia-concept-card-viewer" *ngIf="!loadingMessage">
+  <div class="concept-card-explanation">
+    <div class="oppia-concept-card-explanation-container">
+      <angular-html-bind-wrapper classStr="protractor-test-concept-card-explanation"
+                                 [htmlData]="currentConceptCard.getExplanation().html">
+      </angular-html-bind-wrapper>
+      <div *ngIf="skillDeletedMessage">
+        <div class="oppia-align-center">{{ skillDeletedMessage }}</div>
+      </div>
+    </div>
+  </div>
+  <div class="concept-card-worked-examples" *ngIf="numberOfWorkedExamplesShown > 0">
+    <strong> Worked Examples </strong>
+    <div *ngIf="idx < numberOfWorkedExamplesShown">
+      <div class="oppia-worked-examples-container"
+          *ngFor="let example of currentConceptCard.getWorkedExamples(); let idx = index">
+        <strong> Question: </strong>
+        <angular-html-bind-wrapper classStr="example-content"
+                                   [htmlData]="example.getQuestion().html">
+        </angular-html-bind-wrapper>
+
+        <button class="btn btn-secondary show-example-explanation-button"
+                *ngIf="!explanationIsShown && idx == numberOfWorkedExamplesShown - 1"
+                (click)="explanationIsShown = true;">
+          Show Explanation
+        </button>
+        <div *ngIf="explanationIsShown || idx < numberOfWorkedExamplesShown - 1">
+          <strong> Explanation: </strong>
+          <angular-html-bind-wrapper classStr="example-content"
+                                     [htmlData]="example.getExplanation().html">
+          </angular-html-bind-wrapper >
+        </div>
+      </div>
+    </div>
+    <button class="btn btn-secondary show-more-worked-examples-button"
+            *ngIf="!isLastWorkedExample()"
+            (click)="showMoreWorkedExamples()">
+      + Worked Examples
+    </button>
+  </div>
+</div>
+
+<div *ngIf="loadingMessage">
+  <div class="oppia-align-center">
+    <span [innerHTML]="loadingMessage"></span>
+    <!-- <loading-dots></loading-dots> -->
```

Comment:
Why is this commented?

Reply:
I think that component need not to be used inside other components. It can be used with pages. Angular is unable to load component present at same level therefore removed